tests/_support/Module/TransferModule.php

```diff
@@ -122,7 +141,7 @@ public function assertResolvedXmlHasProperty(
         $simpleXMLElement = $resolvedXml->getSimpleXmlElement();
         $this->assertInstanceOf(SimpleXMLElement::class, $simpleXMLElement);
 
-        $property = $simpleXMLElement->xpath(sprintf('//transfer[@name="%s"]/property[@name="%s"]', $transferName, $propertyName))[0];
+        $property = $simpleXMLElement->xpath(sprintf('//transfer[@name="%s"]/property[@name="%s"]', $transferName, $propertyName))[0] ?? null;
```

Comment:
To provide a more readable error message: "null is not a subclass of SimpleXMLElement" instead of "undefined array offset 0 at ..."